rules/write-a-good-pull-request/rule.md

```diff
@@ -77,27 +77,40 @@ Having the **"What"** information allows the reviewers to quickly understand wha
 
 The PR body is a medium for the developer to tell the reviewers what the changes are about. 
 
-::: info
-**Tip:** For straight-forward changes the self-explanatory title might be enough, so description is not really necessary, but be careful when doing this because the reviewer may not have the same familiarity with the project as you.
+::: greybox
+**PR title:** Update Rule “meaningful-pbi-titles/rule”
+
+**PR description:**  
+:::
+::: bad
+Figure: Bad example - Cannot tell what was done here
 :::
 
-Things that need to be kept in mind before writing a Pull Request body:
+::: greybox
+**PR title:** Update Rule “meaningful-pbi-titles/rule”
 
-* What the PR is about and why did you raise it
-* How the PR will achieve the feature / fix the bug / other goals 
-* (Optional) Include a screenshot if it will help the reviewer to understand the changes (e.g. styling changes)
```

Comment:
For this list item, I think what we had in the other rule is good because it has 2 links:

https://www.ssw.com.au/rules/useful-information-on-changes/#big-and-complex-changes

Reply:
@tiagov8 check if you're happy with how I've included those changes now 🙂